ops/jujuversion.py

```diff
@@ -112,3 +112,8 @@ def is_dispatch_aware(self) -> bool:
     def has_controller_storage(self) -> bool:
         """Determine whether this juju version supports controller-side storage."""
         return (self.major, self.minor, self.patch) >= (2, 8, 0)
+
+    @property
+    def secrets_supported(self) -> bool:
```

Comment:
I suggest we call this `has_secrets` to be consistent with the other names here:

```suggestion
    def has_secrets(self) -> bool:
```